lib/kitchen/directions/move_exercises_to_eoc/v1.rb

```diff
@@ -33,10 +34,11 @@ def bake(chapter:, metadata_source:, klass:, append_to: nil)
       @content = exercise_clipboard.paste
 
       append_to_element = append_to || chapter
-      @tag = append_to ? 'h3' : 'h2'
+      @in_composite_chapter = append_to.present?
       @metadata_title = append_to ? I18n.t(:eoc_exercises_title) : @title
```

Comment:
can this line be removed since this is sorted out in the template

Reply:
Fixed that! And yeah, I think it'll be worth it to have spec for the composite/no composite chapters